.github/workflows/publish.yml

```diff
@@ -13,27 +14,20 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Use Node.js 14 x64
+      - name: Use Node.js 16 x64
         uses: actions/setup-node@v2
         with:
-          node-version: 14
+          node-version: 16
           architecture: x64
           registry-url: "https://registry.npmjs.org"
 
       - name: Install build dependencies
         run: |
           yarn
 
-      - name: publish:release
-        if: github.ref == 'refs/heads/release'
+      - name: Semantic Release
         env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_ACCESS_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Not blocking, but if I'm not mistaken, you shouldn't _need_ this, GitHub gives Actions their own token until this ENV variable, unless you explicitly need a different one with more permissions.

https://docs.github.com/en/actions/security-guides/automatic-token-authentication

Reply:
Ah good call, I kept it in there because I used something like that locally but it probably shouldn't need it here in a github workflow action!